.github/workflows/test_on_push.yml

```diff
@@ -63,18 +49,24 @@ jobs:
     - name: Install liionpack and dependencies
       run: |
         python -m pip install --upgrade pip wheel coverage
-        python -m pip install .
+        python -m pip install ".[dev]"
```

Comment:
Does this need to be `dev`?

Reply:
pytest and nbmake are installed with the dev dependencies, so I changed it here to keep it working. I think the workflows could be broken up a bit and the dependencies could be divided better, but it needs the dev install for now